shoop/core/models/_shops.py

```diff
@@ -46,6 +46,8 @@ class Shop(ChangeProtected, TranslatableShoopModel):
     prices_include_tax = models.BooleanField(default=True, verbose_name=_("prices include tax"))
     logo = FilerImageField(verbose_name=_("logo"), blank=True, null=True, on_delete=models.SET_NULL)
     maintenance_mode = models.BooleanField(verbose_name=_("maintenance mode"), default=False)
+    contact_address = models.ForeignKey(
```

Comment:
Just as a note, this being a mutable address means printouts will be affected "retroactively" by a shop's address changes.

I.e.

* you print something for order X, get address A
* you change your address to B
* you print something for order X again and get address B

I think this is acceptable, just something I wanted to point out :)